var/spack/repos/builtin/packages/openfoam-org/50-etc.patch

```diff
@@ -0,0 +1,24 @@
+--- OpenFOAM-5.x.org/etc/bashrc	2017-07-25 18:43:40.000000000 +0200
++++ OpenFOAM-5.x/etc/bashrc	2017-12-11 13:36:09.479818186 +0100
+@@ -42,15 +42,17 @@
+ #
+ # Please set to the appropriate path if the default is not correct.
+ #
+-[ ${BASH_SOURCE:-$0} ] && \
+-export FOAM_INST_DIR=$(cd $(dirname ${BASH_SOURCE:-$0})/../.. && pwd -P) || \
+-export FOAM_INST_DIR=$HOME/$WM_PROJECT
+-# export FOAM_INST_DIR=~$WM_PROJECT
++rc="${BASH_SOURCE:-${ZSH_NAME:+$0}}"
++[ -n "$rc" ] && FOAM_INST_DIR=$(\cd $(dirname $rc)/../.. && \pwd -P) || \
++FOAM_INST_DIR=$HOME/$WM_PROJECT
+ # export FOAM_INST_DIR=/opt/$WM_PROJECT
+ # export FOAM_INST_DIR=/usr/local/$WM_PROJECT
+ #
+ # END OF (NORMAL) USER EDITABLE PART
+ ################################################################################
++: # Extra safety - if the user commented out all fallback values
++export FOAM_INST_DIR
```

Comment:
This is the final bit of the original `export FOAM_INST_DIR` statement, without it corrupting the logic. Of course, in the near future we can actually ditch this export as well, since it isn't really being respected anymore (this is a holdover from older OpenFOAM versions).